tests/framework/clients/k8s_block.go

```diff
@@ -75,22 +78,23 @@ func (rb *BlockOperation) BlockList() ([]model.BlockImage, error) {
 }
 
 // BlockMap Function to map a Block using Rook
-// Input paramaters -
-//name - path to a yaml file that creates a pod  - pod should be defined to use a pvc that was created earlier
+// Input parameters -
+//name - Pod definition  - pod should be defined to use a pvc that was created earlier
 //mountpath - not used in this impl since mountpath is defined in the pod definition
 //Output  - k8s create pod operation output and/or error
 func (rb *BlockOperation) BlockMap(name string, mountpath string) (string, error) {
-	cmdArgs := []string{name}
-	out, err, status := rb.transportClient.Create(cmdArgs, nil)
-	if status == 0 {
-		return out, nil
+	args := []string{"create", "-f", "-"}
+	result, err := rb.k8sClient.KubectlWithStdin(name, args...)
+	if err != nil {
+		return err.Error(), fmt.Errorf("Unable to map block -- : %s", err)
```

Comment:
In the case of success, i need the std out, error in case of failure. And go doesn't  allow you to return " for nil for string, so I am just returning the error again. 
Also, we are going to update to use the k8s client, so all the kubectl calls will be replaced in future. the func signature will look different.